README.md

```diff
@@ -29,6 +29,7 @@ for more information.
 Install from source
 -------------------
 
+    sudo apt-get install autoconf m4 libtool libtool-bin pkg-config luarocks
```

Comment:
This is specific to Debian-based distributions.  As mentioned a couple lines below, see the wiki page for details.